backend/app/infrastructure/routes/rewards.py

```diff
@@ -243,12 +246,13 @@ def post(self):
     },
 )
 @ns.response(200, "Returns allocation threshold value as uint256")
+@ns.response(400, "Returns when called for an epoch that is not finalized or pending")
```

Comment:
@aziolek it will fail from now on, when called for current and future epochs

Reply:
ACK, thank you for letting us know.
CC: @jmikolajczyk.